test/TestUtilities/Test.Utility/TestDotnetCLiUtility.cs

```diff
@@ -210,7 +211,28 @@ private static void CopyRestoreArtifacts(string artifactsDirectory, string pathT
                 {
                     foreach (FileInfo file in frameworkArtifactsFolder.EnumerateFiles($"*{fileExtension}"))
                     {
-                        file.CopyTo(Path.Combine(pathToSdkInCli, file.Name), overwrite: true);
+                        var dependencyTargetPath = Path.Combine(pathToSdkInCli, file.Name);
+
+                        if (file.Name.StartsWith("NuGet"))
```

Comment:
Shall we add "Microsoft.Build.NuGetSdkResolver" at here?

Reply:
Great point. I had contains originally and then I removed it :D